appengine/datastore/app.rb

```diff
@@ -18,14 +18,15 @@
 require "google/cloud/datastore"
 
 get "/" do
+  datastore = Google::Cloud::Datastore.new
+
   # Save visit in Datastore
-  visit = dataset.entity "Visit"
+  visit = datastore.entity "Visit"
```

Comment:
Out of scope and okay to decline but it would make the sample cleaner.
[Example](http://googlecloudplatform.github.io/google-cloud-ruby/docs/google-cloud-datastore/v1.4.3/Google/Cloud/Datastore/Dataset.html#save-instance_method) shows how to use a Ruby block which gives context to the entity visit when setting `user_ip` and `timestamp`.

Reply:
Done.